specification/dimensions/commitmentdiscounttype.md

```diff
@@ -0,0 +1,39 @@
+# Commitment Discount Type
+
+A commitment-based discount is a contractual commitment in exchange for a discounted unit price. Commitment discounts are available as spend-based or resource-based discounts, depending on the CSP offering them and the particular service.
+
+This column determines whether the commitment is based on a minimum spend or minimum usage.
```

Comment:
nit: "Spend" is jargon. Do we want to use that? We haven't taken a stance on whether we'll use jargon or not. Admittedly, "cost" doesn't roll off the tongue in the same way, but we do have "cost" columns and we do not have "spend" columns. Food for thought...

Reply:
If Spend and Cost can be used interchangeably then it's probably wise it standardise on Cost and not introduce another term. Anyone else have thoughts on this?

Reply:
Tough call. `Spend` here is part of a descriptor of a commitment, representing a commitment of monetary-amount per time-unit. Is the question, "Do we allow jargon in field values?" 

If we standardised on "cost": the "cost" **_at the basis of the_** commitment is a different term from "cost" **_of the_** commitment. Not clear to me yet if this is problematic.

Reply:
I feel this needs to be part of our core design principles. Let me bring this to the practitioner squad and see what they say. Leave this comment open, but let's not block on it. If anyone has any opinions, please share them here.

Reply:
The word "spend" is used by AWS ("Savings Plans is a flexible pricing model that can help you reduce your bill by up to 72% compared to On-Demand prices, in exchange for a one- or three-year hourly spend commitment"), by GCP ("Spend-based committed use discounts provide a discount in exchange for your commitment to spend a minimum amount for a product or service."), and by Azure ("[Savings Plan] Get lower prices by agreeing to spend a fixed hourly amount on compute services for one or three years.").

Moreover, "spend-based commitment discount" sounds much clearer than "cost-based commitment discount".

Reply:
Ok, going with Spend